docs/integrations/gateways/aws-api-gateway.mdx

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_position: 2
+title: AWS API Gateway x Permit
+---
+
+This document describes how to create an API Gateway that uses a Permit authorizer to authenticate requests.
+
+## Create a Permit Authorizer
+In order to create an Authorizer that uses Permit, you will need to create a Lambda function that uses Permit to authenticate requests.
+
+### Create a Lambda Function
+First, create a new Lambda function - we use Python 3.12 in this example but you can use any runtime that Permit supports with one
+of our [SDKs](/sdk/sdks-overview)
+
+![Lambda creation](/img/integrations/aws/aws-1.png)_
+
+### Create a Layer
```

Comment:
Add a small description that describe this is required to be in local environment and that you need venv and python installed.

Reply:
done